src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -607,5 +584,9 @@ public class SimpleEvents {
 					"\t\tsend \"Oops! Mending failed!\" to player")
 				.since("2.5.1");
 		}
+		Skript.registerEvent("Advancement Grant", SimpleEvent.class, PlayerAdvancementDoneEvent.class, "advancement [grant]")
```

Comment:
This syntax needs improvement. "Grant" suggests it's being given (which it could be) but that event is for finishing all criteria.
Maybe try a word like complete/finish/get instead.